mne_connectivity/spectral/time.py

```diff
@@ -9,46 +9,50 @@
 from mne.parallel import parallel_func
 from mne.time_frequency import (tfr_array_morlet, tfr_array_multitaper,
                                 dpss_windows)
-from mne.utils import (logger, warn, verbose)
+from mne.utils import (logger, verbose)
 
 from ..base import (SpectralConnectivity, EpochSpectralConnectivity)
-from .epochs import _compute_freqs, _compute_freq_mask
+from .epochs import _compute_freq_mask
 from .smooth import _create_kernel, _smooth_spectra
 from ..utils import check_indices, fill_doc
 
 
 @verbose
 @fill_doc
-def spectral_connectivity_time(data, method='coh', average=False,
+def spectral_connectivity_time(data, freqs, method='coh', average=False,
                                indices=None, sfreq=None, fmin=None,
                                fmax=None, fskip=0, faverage=False, sm_times=0,
-                               sm_freqs=1, sm_kernel='hanning',
+                               sm_freqs=1, sm_kernel='hanning', padding=0,
                                mode='cwt_morlet', mt_bandwidth=None,
-                               cwt_freqs=None, n_cycles=7, decim=1,
-                               n_jobs=1, verbose=None):
-    """Compute frequency- and time-frequency-domain connectivity measures.
+                               n_cycles=7, decim=1, n_jobs=1, verbose=None):
+    """Compute time-frequency-domain connectivity measures.
 
-    This method computes time-resolved connectivity measures from epoched data.
+    This function computes spectral connectivity over time from epoched data.
+    The data may consist of a single epoch.
 
     The connectivity method(s) are specified using the ``method`` parameter.
-    All methods are based on estimates of the cross- and power spectral
-    densities (CSD/PSD) Sxy and Sxx, Syy.
+    All methods are based on time-resolved estimates of the cross- and
+    power spectral densities (CSD/PSD) Sxy and Sxx, Syy.
 
     Parameters
     ----------
     data : array_like, shape (n_epochs, n_signals, n_times) | Epochs
         The data from which to compute connectivity.
+    freqs : array_like
```

Comment:
It can be `int`, or `float` right? If so, what is the expected behavior? It just computes the connectivity at that frequency?